src/Server.ts

```diff
@@ -1,5 +1,7 @@
 import * as express from 'express';
+import * as bodyParser from 'body-parser';
 import Iconfig from './config/iconfig';
+import { errorHandler, notFoundRoute } from './libs/routes/index';
```

Comment:
remove index from the path, no need to add index while importing

Reply:
Done